src/cpp/core/include/core/rapidxml/rapidxml.hpp

```diff
@@ -1327,7 +1327,7 @@ namespace rapidxml
     
         // Note that some of the pointers below have UNDEFINED values if certain other pointers are 0.
         // This is required for maximum performance, as it allows the parser to omit initialization of 
-        // unneded/redundant values.
+        // unneeded/redundant values.
```

Comment:
I'd recommend making these changes upstream; this is a 3rd-party component.

Reply:
Reverted in 52dc3e8